ts/store/reducers/entities/services/index.ts

```diff
@@ -63,6 +64,35 @@ const reducer = combineReducers<ServicesState, Action>({
 // Selectors
 export const servicesSelector = (state: GlobalState) => state.entities.services;
 
+// A selector to get the organizations selected by the user as areas of interests
+export const organizationsOfInterestSelector = createSelector(
+  [userMetadataSelector, servicesSelector],
+  (potUserMetadata, services) => {
+    const visibleServices = new Set(
```

Comment:
maybe the lint error is due to this
```suggestion
    const visibleServices = new Set<string>(
```

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1327/commits/637b2fc1a624718269cd7320b95619391572c1d9